pkg/etcd/s3.go

```diff
@@ -199,7 +255,7 @@ func (s *S3) snapshotRetention(ctx context.Context) error {
 
 	delCount := len(snapshotFiles) - s.config.EtcdSnapshotRetention
 	for _, df := range snapshotFiles[:delCount] {
-		logrus.Debugf("Removing snapshot: %s", df.Key)
+		logrus.Infof("Removing S3 snapshot: %s", df.Key)
```

Comment:
Was this turned to Info from Debug for testing and if so, should it be changed back?

Reply:
It was set to info so as to help provide as much transparency to the user regarding what we are removing

We are greatly increasing the "logging" of the `etcd-snapshot` command through this PR and I did my best to selectively provide feedback to the user that is considered useful (and in my opinion, knowing what we are removing is useful)